README.md

```diff
@@ -82,16 +95,13 @@ const { image } = await preview({
         width: 200,
         height: 200
     },
-    getTile: function (x, y, z, callback) {
-        renderer.getTile('png', z, x, y)
-            .then(({ tile }) => callback(null, tile))
-            .catch((err) => callback(err))
+    getTile: async function (format, x, y, z) {
+        const { buffer } = await renderer.getTile(format, z, x, y)
+        return { buffer }
```

Comment:
I would say `return buffer` for the sake of simplicity

Reply:
I'm afraid we can't do it. As per https://github.com/CartoDB/cartonik/blob/1b22ad88765af8cad3b2f043d43d3601cc751004/lib/preview/blend.js#L50 the signature of the function `getTile` must be:  `async getTile(format, z, x, y) -> Promise<{ buffer, headers, stats }|Error>.

Reply:
Yeah, you are right, I forget I have written that